python/flatbuffers/util.py

```diff
@@ -20,6 +20,14 @@ def GetSizePrefix(buf, offset):
 	"""Extract the size prefix from a buffer."""
 	return encode.Get(packer.int32, buf, offset)
 
+def GetBufferIdentifier(buf, offset, size_prefixed = False):
+        """Extract the file_identifier from a buffer"""
+        offset+=(number_types.UOffsetTFlags.bytewidth if size_prefixed else 0) + number_types.UOffsetTFlags.bytewidth # offset sizeof(root table offset) + sizeof(size prefix) if present   
```

Comment:
Why is this `bytewidth`?

Reply:
`number_types.*Flags.bytewidth` defines the size, in bytes, of that type. Acording to the standard, the root table offset is a `uoffset_t`

Reply:
The standard says that currently this is `uint32_t`, implying this could change.

Reply:
Right, I misread :-)